src/welcome_notification.js

```diff
@@ -1,73 +0,0 @@
-// show welcome message on first installation
-const path = require('path')
-const fs = require('fs')
-
-const WELCOME_SHOWN = path.join(__dirname, '..', 'WELCOME_SHOWN')
-
-const generateWelcomeMsg = () => {
-  return `## Welcome to ide-html\n\nQuick start for ide-html:\n${[
-    'Open a HTML file from the Project',
-    'Dispatch command `Outline View: Toggle`',
-  ].map((str, i) => `${i}. ${str}\n`).join('')
-  }\nHappy Using Atom IDE : )`
-}
-
-const generateWarnMsg = () => {
-  return `## Welcome to ide-html\n\nPlease install following requirements:\n${[
-    '[`atom-ide-ui`](https://atom.io/packages/atom-ide-ui)',
-  ].map((str, i) => `${i}. ${str}\n`).join('')
-  }\nso that ide-html can work properly`
-}
-
-const helpButton = [
-  {
-    text: ' Help',
-    onDidClick: () => { atom.commands.dispatch(
-      atom.views.getView(atom.workspace.getActivePane()),
-      'ide-html:help'
-    )},
-    className: 'btn btn-success btn-lg icon-light-bulb',
-  },
-]
-
-const settingsButton = [
-  {
-    text: ' Open Settings Page',
-    onDidClick: () => {
-      atom.workspace.open('atom://config/install/package:atom-ide-ui')
-    },
-    className: 'btn btn-success btn-lg icon-tools',
-  },
-]
-
-const showWelcomeNotification = () => {
-  try {
-    fs.accessSync(WELCOME_SHOWN)
-  } catch (err) {
-    if (atom.packages.isPackageLoaded('atom-ide-ui')) {
```

Comment:
If we look at other packages such as `ide-typescript` and `ide-php` they don't tell the user to install the `atom-ide-ui` package. I think it would be good if these packages also followed the conventions.

With that said, I see your point about telling the user about `atom-ide-ui`. Maybe we should look at the [package-deps](https://github.com/steelbrain/) package which helps with this?

Reply:
Nice. Put some notes here
https://www.npmjs.com/package/atom-package-deps